cfme/tests/configure/test_access_control.py

```diff
@@ -1119,6 +1119,65 @@ def test_user_change_password(appliance, request):
         assert appliance.server.current_full_name() == user.name
 
 
+def test_tenant_quota_input_validate(appliance):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Infra
+        initialEstimate: 1/8h
+    """
+    roottenant = appliance.collections.tenants.get_root_tenant()
+    fields = [('cpu', 2.5), ('storage', '1.x'), ('memory', '2.x'), ('vm', 1.5)]
+
+    for field in fields:
+        view = navigate_to(roottenant, 'ManageQuotas')
+        view.form.fill({'{}_cb'.format(field[0]): True, '{}_txt'.format(field[0]): field[1]})
+        assert view.save_button.disabled
+        view.form.fill({'{}_cb'.format(field[0]): False})
+
+
+def test_delete_default_tenant(appliance):
+    """
+    Steps:
+    1. Login as an 'Administrator' user
+    2. Navigate to configuration > access control > tenants
+    3. Select default tenant('My Company') from tenants table
+    4. Delete using 'configuration > Delete selected items'
+    5. Check whether default tenant is deleted or not
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Infra
+        initialEstimate: 1/20h
+    """
+    view = navigate_to(appliance.collections.tenants, "All")
+    roottenant = appliance.collections.tenants.get_root_tenant()
+    msg = 'Default Tenant "{}" can not be deleted'.format(roottenant.name)
+    tenant = appliance.collections.tenants.instantiate(name=roottenant.name)
+    appliance.collections.tenants.delete(tenant)
+    assert view.flash.assert_message(msg)
+    assert roottenant.exists
+
+
+def test_copied_user_password_inheritance(appliance, group_collection, request):
+    """Test to verify that dialog for copied user should appear and password field should be
+    empty
+
+    Polarion:
+        assignee: anikifor
+        initialEstimate: 1/15h
```

Comment:
@mnadeem92 Please add `casecomponent` of respected test.

Reply:
@lina-nikiforova as your are owner of this test. can you please suggest case-component for this test to Nadeem?

Reply:
I suggest `casecomponent` to be `WebUI`.
Also in polarion it's `caseimportance: high`. Would be nice if you add it here :)